Woof/Woof/Helper/AppAlert/AppAlert.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Represents alerts which uses in application.
+enum AppAlert: String, CustomStringConvertible {
+    /// Alert shown as confirmation of log out
+    case logOut = "Do you really want to log out?"
+
+    /// Alert shown as a result of an error
+    case error = "Error"
+
+    /// Text description of alert
+    var description: String {
+        rawValue
+    }
+}
```

Comment:
Let's create the new PR for this enhancement, as we need to add unit tests for new enums and the current PR is already so big.

Reply:
Fixed with 19ade1f